### ISSUE TITLE
API to lookup process by name (string)

### ISSUE THREAD

**Issue by soxofaan** [Open-EO/openeo-processes-python#10]
It would be handy to be able to retrieve a process function by just providing its name as a string.
For example for when you are programmatically evaluating a given process graph.

Also nice to have in this context: retrieve the schema of the process parameters and return value

**Comment by lforesta** [comment]
so you would need to have each process in its own file, so that you can do a GET request to a URL containing the process name and get the code? 

For the schema, you can already do that, e.g. GET https://github.com/Open-EO/openeo-processes/blob/master/absolute.json

**Comment by soxofaan** [comment]
no sorry I was not very clear in my question, I didn't mean a HTTP API, just functionality in the python package.

Something like

```python
import openeo_processes

fun = openeo_processes.get_process("add")

fun(1,3)
# -> 4
```

**Comment by claxn** [comment]
Thanks @soxofaan , that looks indeed very helpful to me!

**Comment by soxofaan** [comment]
I think this can be closed as #13 is merged
